app/src/main/java/mozilla/lockbox/view/AccountSettingFragment.kt

```diff
@@ -42,11 +41,16 @@ class AccountSettingFragment : BackableFragment(), AccountSettingView {
     }
 
     override fun setAvatar(avatar: Avatar) {
+        val url =
+            if (avatar.isDefault) {
+                null
+            } else {
+                avatar.url
+            }
+
         Picasso.get()
-            .load(avatar.url)
+            .load(url)
             .placeholder(R.drawable.ic_avatar_placeholder)
-            .resize(80, 80)
-            .transform(CropCircleTransformation())
```

Comment:
can we delete our custom `CropCircleTransformation` now? or do we use it somewhere else?

Reply:
This is a Picasso file, not ours

Reply:
no, it comes from the `thirdparty` directory in our app (specifically `thirdparty/src/main/java/jp/wasabeef/picasso/transformations/CropCircleTransformation.kt`).